ibis/backends/polars/__init__.py

```diff
@@ -224,8 +239,10 @@ def create_table(
         overwrite: bool = False,
     ) -> ir.Table:
         if schema is not None and obj is None:
-            raise NotImplementedError(
-                "Empty table creation is not yet supported in the Polars backend"
+            import pyarrow as pa
+
+            obj = pl.LazyFrame(
+                pa.Table.from_pylist([], schema=schema_to_pyarrow(schema))
```

Comment:
You can pass a schema to `LazyFrame` directly, then `pl.LazyFrame([], schema=polars_schema)` should be sufficient without importing pyarrow.

Reply:
Nice 🎉